### PR TITLE
[DEP][SP] Move common functions to utils.js and include it in samples

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
BUG=XWALK-6342

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@jondong @huningxin Please help to review. I can't verify the changes in DEP, because DEP is not work in my ENV. I tried the newst master branch, Debug&Release versions. @jondong Please help to check, Thanks!

**Comment by DonnaWuDongxia** [comment]
dep-failed-small

**Comment by huningxin** [comment]
@DonnaWuDongxia , you may miss the "utils.js" file.

File: sample/src/enhancedphotography.js [hunk 1]
```diff
@@ -419,9 +407,9 @@ function main() {
     }
     XDMUtils.saveXDM(savePhoto).then(
         function(blob) {
-          xwalk.experimental.native_file_system.requestNativeFileSystem('pictures',
+          RSUtils.nativeFS.requestNativeFileSystem('pictures',
```
Comment (huningxin) [inline sample/src/enhancedphotography.js]: I would suggest not to wrap Crosswalk API. I am fine to use alias for short name. But in a explicit way. One important purpose of our sample is to showcase the API usage.

Comment (jondong) [inline sample/src/enhancedphotography.js]: +1. it seems there's no necessity to do this.

**Comment by jondong** [comment]
@DonnaWuDongxia your patch won't work for me cause you've missed the utils.js.

**Comment by DonnaWuDongxia** [comment]
@huningxin @jondong updated, thanks!

File: sample/src/utils.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    var dateString =
+        date.getFullYear() +
+        ('0' + (date.getMonth() + 1)).slice(-2) +
+        ('0' + date.getDate()).slice(-2) +
+        ('0' + date.getHours()).slice(-2) +
+        ('0' + date.getMinutes()).slice(-2) +
+        ('0' + date.getSeconds()).slice(-2);
+    return dateString;
+  }
+};
```
Comment (jondong) [inline sample/src/utils.js]: This file's line ending seems odd to me(although it cannot be seen here), it seems we'd submit the patch with Unix-like line endding format.

**Comment by jondong** [comment]
@DonnaWuDongxia Patch verified and it works fine for me. LGTM as soon as you fixed the CRLF issue of utils.js.

**Comment by DonnaWuDongxia** [comment]
@jondong Good capture, how can you find it? It shows no difference by git diff.

**Comment by DonnaWuDongxia** [comment]
@jondong updated. I removed them using tr in cygwin.

**Comment by jondong** [comment]
LGTM. merging.

**Pull request merged**